content/200-orm/200-prisma-client/500-deployment/301-edge/100-overview.mdx

```diff
@@ -0,0 +1,58 @@
+---
+title: 'Overview'
+metaTitle: 'Overview (Deploy Prisma ORM at the Edge)'
+metaDescription: 'Learn how to deploy your Prisma-backed apps to edge functions like Cloudflare Workers or Vercel Edge Functions'
+tocDepth: 2
+---
+
+## Deploying edge functions with Prisma ORM
+
+You can deploy an application that uses Prisma ORM to the edge. Depending on which edge function provider and which database you use, there are different considerations and things to be aware of.
+
+Here is a brief overview of all the edge function providers that are currently supported by Prisma ORM:
+
+| Provider / Product      | Supported natively with Prisma ORM                      | Supported with Prisma Accelerate |
+| ----------------------- | ------------------------------------------------------- | -------------------------------- |
+| Vercel Edge Functions   | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Vercel Edge Middleware  | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Cloudflare Workers      | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Cloudflare Pages        | ✅ (Preview; only compatible drivers)                   | ✅                               |
+| Deno Deploy             | [Not yet](https://github.com/prisma/prisma/issues/2452) | ✅                               |
+| Netlify Edge Functions  | Not yet                                                 | ✅                               |
+| Supabase Edge Functions | Not yet                                                 | ✅                               |
```

Comment:
Are we sure you can actually deploy Prisma to Netlify and Supabase Edge Functions?

Reply:
It was my understanding that using Accelerate, we can deploy to all Edge function providers. I haven't done any tests, so if you're saying it's not the case I can remove these.

Reply:
Related discussion: https://prisma-company.slack.com/archives/C04F60M3G06/p1710237616684739